src/import/import-compress.c

```diff
@@ -293,19 +294,28 @@ int import_compress_init(ImportCompress *c, ImportCompressType t) {
 #if HAVE_ZSTD
         case IMPORT_COMPRESS_ZSTD: {
                 size_t r2;
+                int level = ZSTD_CLEVEL_DEFAULT;
                 int ncpus;
+                int64_t env;
 
                 assert(c->zstd_c == NULL);
                 c->zstd_c = ZSTD_createCStream();
                 if (c->zstd_c == NULL)
                         return -ENOMEM;
 
+                r = getenv_int64("SYSTEMD_IMPORT_COMPRESS_LEVEL", &env);
```

Comment:
this is a new env var. should be documented in docs/ENVIRONMENT.md

Reply:
also, the env var name should have a suffix of `_ZSTD` or so. compression level ranges are specific to the compression algorithm after all, and hence a generic variable covering them all doesn't really make sense.